pages/docs/styled-system/theming/theme.mdx

```diff
@@ -267,7 +267,7 @@ const spacing = {
   },
 }
 
-const theme = extendTheme({ spacing, ... })
+const theme = extendTheme({ space: spacing.space, ... })
```

Comment:
Instead of doing `space: spacing.space`, you can just spread the `spacing` variable,
```suggestion
const theme = extendTheme({ ...spacing })
```